packages/context/README.md

```diff
@@ -160,7 +160,16 @@ Returns a function that can be used to navigate to another page. The returned fu
 
 #### `useApp()`
 
-Returns the currently loaded Evoke app.
+Returns an object with the following properties:
```

Comment:
The previous description is still accurate, it is still returning the Evoke app.  I think the new description ("an object") makes it more nebulously what it returns, and also it's not accurate; it does not have an `app` property.